sympde/expr/tests/test_expr.py

```diff
@@ -1891,6 +1895,200 @@ def test_interface_integral_4():
     assert expr[0].expr[1,1]   == u2[1]*v2[1]
 
     # ...
+
+
+#==============================================================================
+def test_matrices_vectors():
+
+    domain = Domain('Omega', dim=3)
+    x,y,z = domain.coordinates
+
+    kappa = Constant('kappa', is_real=True)
+    mu    = Constant('mu',    is_real=True)
+    theta = Constant('theta',   is_real=True)
+    a1    = Constant('a1',    is_real=True)
+    a2    = Constant('a2',    is_real=True)
+    b1    = Constant('b1',    is_real=True)
+    b2    = Constant('b2',    is_real=True)
+
+    V = ScalarFunctionSpace('V', domain)
+    W = VectorFunctionSpace('W', domain)
+
+    u,v   = [element_of(V, name=i) for i in ['u', 'v']]
+    F,G   = [element_of(W, name=i) for i in ['F', 'G']]
```

Comment:
Shorter and more elegant:
```suggestion
    u, v   = elements_of(V, names='u, v')
    F, G   = elements_of(W, names='F, G')
```